core/src/main/java/io/grpc/internal/AutoConfiguredLoadBalancerFactory.java

```diff
@@ -125,48 +120,53 @@ Status tryHandleResolvedAddresses(ResolvedAddresses resolvedAddresses) {
       }
       PolicySelection policySelection =
           (PolicySelection) resolvedAddresses.getLoadBalancingPolicyConfig();
-      ResolvedPolicySelection resolvedSelection;
```

Comment:
can you also remove the `ResolvedPolicySelection` class?

Reply:
Done.